src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/InstantiateArgumentExceptionsCorrectly.cs

```diff
@@ -121,13 +121,19 @@ public override void Initialize(AnalysisContext analysisContext)
                         break;
                     }
                 }
+
                 if (diagnostic != null)
                 {
                     context.ReportDiagnostic(diagnostic);
                 }
             }
         }
 
+        private static bool MatchesConfiguredVisibility(ISymbol owningSymbol, OperationAnalysisContext context)
+        { // Should add for all rules?
```

Comment:
```suggestion
        {
            // Should add for all rules?
```

Reply:
Oh it was comment for myself, the `MatchesConfiguredVisibility(...)` method taking DiagnosticRule argument but we have 3 rules here, so I was wondering if I should check for all rules, but that was not needed, I have test for other rule, it was working, will remove